iohk/test/UpdateLogicSpec.hs

```diff
@@ -4,22 +4,26 @@ module UpdateLogicSpec (spec, main) where
 
 import Test.Hspec hiding (context)
 import qualified Data.ByteString.Lazy as LBS
+import qualified Data.Text as T
+import Data.Aeson (decode)
 
 import UpdateLogic
-import Github (CommitStatus(..), Status(..))
+import Github (CommitStatus(..), Status(..), GitHubSource(..))
 import qualified Buildkite.API as BK
 
 main :: IO ()
 main = hspec spec
 
-sampleInput :: LBS.ByteString
-sampleInput = "junk\nbuild id is 13711'\r\r\njunk"
+sampleCardanoSLSrc = "{ \"url\": \"https://github.com/input-output-hk/cardano-sl\", \"fetchSubmodules\": \"true\", \"rev\": \"1bba2fd0183f575752f4529182205e04074db336\", \"sha256\": \"1qfi307x7nfp42ldb6sjx30mk7b0zfvd4b48pmyd9g465f7sp420\" }" :: LBS.ByteString
 
 spec :: Spec
 spec = do
   describe "travis log output" $ do
```

Comment:
should this be something else now that we aren't using travis for CI tests anymore? Maybe CI log output?

Reply:
FYI, don't consider this blocking, since you aren't touching it, but maybe we open new YT issue for it?